### ISSUE TITLE
Thaumcraft Stabilizer Crashes Server

### ISSUE THREAD

**Issue by Ameseree** [AllTheMods/ATM-3#655]
As mentioned in the title, when the thaumcraft stabilizer is used the server crashes and is unable to start back up.

## Expected Behavior
The stabilizer is suppose to stabilize thaumcraft devices and entities such as the flux rift.

## Current Behavior
Upon powering the stabilizer the server crashes

## Possible Solution
The newest version of thaumcraft has fixed this issue I would update to it as soon as possible.

## Steps to Reproduce
1. Be on a server. Otherwise it won't crash.
2. Make thaumcraft stabilizer
3. Point it towards a flux rift
4. Power it.

## Log
<script src="https://gist.github.com/Ameseree/c21de0788c72152693bef25040d9ed98.js"></script>
I know it's a slightly older version of the modpack but the issue is related to the version of thaumcraft and not the version of the modpack.

## Detailed Description
Basically just need to update thaumcraft from B16 to B17.

**Comment by FallenEagle1** [comment]
Known issue, will be fixed when the pack next updates.
